src/cryptography/hazmat/primitives/padding.py

```diff
@@ -42,7 +42,9 @@ def _byte_padding_update(buffer_, data, block_size):
 
     utils._check_byteslike("data", data)
 
-    buffer_ += bytes(data)
+    # Only coerce non-bytes objects to avoid triggering bad behavior of future's
+    # newbytes type. Unconditionally call bytes() after Python 2 support is gone.
```

Comment:
Can you put words `six.PY2` in this comment so we'll find it with grep when teh time comes.